### DISCUSSION TITLE
Does anyone know the keycode/file path to G-Helper Ultimate Mode? Looking to assign a keyboard shortcut to this

### DISCUSSION THREAD

**Discussion by Zackeroo** [seerge/g-helper#2702] [Q&A]
I've tried to find a file path to enable this but had no luck. Was hoping there might be a keycode that could do this too... If there isn't a way I'll just req this as a new feature as this could be quite useful for a lot of people. FN + C command is being unused currently and mapping this to that shortcut would be perfect. 

Thanks!

**Comment by seerge** [comment]
@Zackeroo hello, there is no hotkey for that cause Ultimate toggle requires reboot anyway.
